src/dbnode/storage/bootstrap/bootstrapper/readers.go

```diff
@@ -110,15 +133,28 @@ func newShardReaders(
 	readerPool *ReaderPool,
 	shard uint32,
 	tr xtime.Ranges,
+	dataReaderDoNotSort bool,
 	logger *zap.Logger,
+	span opentracing.Span,
+	nowFn clock.NowFn,
 ) ShardReaders {
+	start := nowFn()
+	logger.Debug("enqueue readers read info files start")
+	span.LogEvent("enqueue_readers_read_info_files_start")
 	readInfoFilesResults := fs.ReadInfoFiles(fsOpts.FilePathPrefix(),
 		ns.ID(), shard, fsOpts.InfoReaderBufferSize(), fsOpts.DecodingOptions(), persist.FileSetFlushType)
+	logger.Debug("enqueue readers read info files done",
+		zap.Duration("took", nowFn().Sub(start)))
+	span.LogEvent("enqueue_readers_read_info_files_done")
+
 	if len(readInfoFilesResults) == 0 {
 		// No readers.
 		return ShardReaders{}
 	}
 
+	start = nowFn()
+	logger.Debug("enqueue readers open data readers start")
+	span.LogEvent("enqueue_readers_open_data_readers_start")
```

Comment:
For these span log events (and maybe debug log messages) want to add the `shard` and `tr.Summary()` (time range summary string) as tags?